apis/apis.json

```diff
@@ -18,6 +18,23 @@
       "restNumericEnums": true,
       "transport": "grpc+rest"
     },
+    {
+      "id": "Google.Cloud.AspNetCore.DataProtection.SecretManager",
+      "version": "1.0.0-alpha01",
+      "targetFrameworks": "netstandard2.1",
+      "type": "other",
+      "description": "Recommended Google client library to use Google Cloud Secret Manager to store data protection keys for ASP.NET Core.",
+      "tags": [
+        "Secret Manager",
+        "Google",
+        "Cloud",
+        "Security"
+      ],
+      "dependencies": {
+          "Google.Cloud.SecretManager.V1": "2.1.0",
```

Comment:
Nit: unindent to two spaces. (I suspect that running generateprojects.sh will do that for you automatically.)